appveyor.yml

```diff
@@ -1,23 +0,0 @@
-environment:
```

Comment:
For some reason appveyor is still trying to build- then failing. Might want to figure out how to disable that.

Reply:
Appveyor is installed into this github repository, so we'll just have to remove that from the repo's control panel.